astropy/coordinates/matrix_utilities.py

```diff
@@ -136,14 +136,18 @@ def angle_axis(matrix):
     return Angle(angle, u.radian), -axis / r
 
 
-def is_O3(matrix):
+def is_O3(matrix, atol=1e-15):
```

Comment:
```suggestion
def is_O3(matrix, atol=None):
```

Reply:
@nstarman, I changed a bit your suggestion, as one of the tests uses an int64 matrix and ``finfo`` was giving me the numpy error
```
ValueError: data type <class 'numpy.int64'> not inexact
```
Once fixed it, the value of ```atol``` for integers was ```None``` and that was giving me another numpy error
```
TypeError: unsupported operand type(s) for +: 'NoneType' and 'float'
```
I hope the current version reproduces the expected behavior...